README.md

```diff
@@ -1,58 +1,54 @@
 # Micromasters
 Portal for learners and course teams to access MITx Micromasters programs
 
-## Getting Started
-
-Although you can run Micromasters locally with a default sqlite database after
-installing the ``requirements.txt`` file, the recommended way is to 
-use [Docker](https://www.docker.io). Install Docker, then install 
-``docker-compose`` and run the app: 
+## Major Dependencies
+- pip, virtualenv, homebrew (OSX only)
+- Postgres >= 9.4
+  - OSX recommended install method: [Postgres.app](http://postgresapp.com/)
+- Python 3
+  - OSX recommended install method: Homebrew `brew install python3`
+- Docker (_optional, but strongly recommended_)
+  - OSX recommended install method: [Download from Docker website](https://docs.docker.com/mac/)
+- docker-compose (_if you're using Docker, this is required_)
+  - Recommended install: pip `pip install docker-compose`
 
-    pip install docker-compose 
-    docker-compose up
+## Getting Started
 
-### (OSX only) Create your docker machine:
+1. Create and activate a virtualenv in the root folder of the Micromasters project directory.
+1. Install the python dependencies ``pip install -r requirements.txt``
 
-    docker-machine create default
-    docker-machine start default
-    docker-machine env default
-    eval "$(docker-machine env default)"
+#### (OSX only) Create your docker container:
 
-These commands create a Docker container named ``default``, start the
+These commands create a Docker container named ``mm``, start the
 container, and configure environment variables to facilitate communication
-with the ``edX`` instance.
+with the edX instance.
 
+    docker-machine create mm
+    docker-machine start mm
+    # 'docker-machine env (container)' prints export commands for important environment variables
+    eval "$(docker-machine env mm)"
```

Comment:
Curious about the recommendation to name it mm instead of default. Makes sense I guess.